### PR TITLE
ENH: Beam statistics plan, among other things

### PR DESCRIPTION
**Pull request by apra93**
*TL;DR* - 
- Beam statistics plan added
- Measure() takes max_dropped as arg
- HX2 and DG3 have images
- Generalized as_list to handle tuples
- Centroid y is returns center of y in image
- landscape.yml added
- Deprecated filters.py
- Tests

The core functionality introduced with this PR is a plan that calculates statistics which should characterize the profile of the beam. It takes a list of detectors, the image field, image size field, and centroid fields (fields are strings) along with a number of image processing arguments and then returns a dictionary of the calculated beam statistics. The processing done on the image and the centroid can be returned independently that way we don't have to run the image processing pipeline each time. 

Measure() now takes the additional argument max_dropped, which specifies the number of events that can be filtered before raising an error. This was previously hard-coded to be 50.

Another addition is that the detector objects (hx2 and dg3) now have mocked image returning signals which was used to do the testing for the statistics, but can be used elsewhere. A planned improvement is to make the image centroid match the calculated centroid by rolling the numpy array - for now, it just returns an image from the hx2 or dg3 folders in psbeam.

Slight modifications were made in a couple other places: as_list was made to be slightly more general so it can handle tuples the right way (or any iterable that isn't a list) but doesn't break the existing functionality. 

Centroid y previously just returned 0 but will now return half the size of the image, or the center of the image in y.

I also added the landscape.yml file along with the badge to the readme. This will at worst do nothing for us, and at best makes it clear(er) how much technical debt we are in with the module by displaying what flake8/pylint see as problematic that our tests do not.

Filters.py was removed because the filters are all in psbeam and are tested there.

### CODE REVIEW

File: pswalker/plans.py [hunk 1]
```diff
@@ -255,6 +256,9 @@ def measure(detectors, num=1, delay=None, filters=None, drop_missing=True):
     drop_missing : bool, optional
         Choice to include events where event keys are missing
 
+    max_drops : int, optional
```
Comment (ZLLentz) [inline pswalker/plans.py]: The docstring added here has the wrong keyword: max_drops instead of max_dropped

**Comment by ZLLentz** [comment]
Other than my minor docstring comment, looks good to me. If you're out this week, I'll fix the doc line and merge it in later today.
